packages/lesswrong/lib/annualReviewMarkets.ts

```diff
@@ -0,0 +1,23 @@
+export const PROBABILITY_REVIEW_WINNER_THRESHOLD = 0.5
+
+export const MINIMUM_KARMA_REVIEW_MARKET_CREATION = 100;
+
+export type AnnualReviewMarketInfo = {
+  probability: number;
+  isResolved: boolean;
+  year: number;
+}
+
+export const getMarketInfo = (post: PostsBase): AnnualReviewMarketInfo | null => {
+  if (post.annualReviewMarketProbability == null) return null
+  if (post.annualReviewMarketIsResolved == null) return null
+  if (post.annualReviewMarketYear == null) return null
```

Comment:
Is there a reason to have `==` rather than `===` here?  If any of them can actually be `undefined` rather than just `null` I'd prefer an explicit check (either checking against both, or checking e.g. `typeof post.annualReviewMarketProbability !== 'number'`, etc.)